main.js

```diff
@@ -0,0 +1,19 @@
+function daysInYear(year) {
+  const errorMessagge = 'exception';
```

Comment:
Why you declare it here, maybe you do not need it?
Move outside the function as constant, or declare in catch block